src/structures/Realm.js

```diff
@@ -44,4 +44,12 @@ module.exports = class Realm {
   async close () {
     return this.#api.changeRealmState(this.id, 'close')
   }
+
+  async getWorldDownload () {
+    return this.#api.getRealmWorldDownload(this.id, this.activeSlot, 'latest')
```

Comment:
what is activeSlot ? I don't see it in the docs

Reply:
Realms have 3 slots for worlds, activeSlot is the currently selected slot and is a property of `Realm`